### PR TITLE
feat: Add JSDoc comments to output

### PR DESCRIPTION
**Pull request by ExE-Boss**
This&nbsp;makes writing&nbsp;Web&nbsp;API&nbsp;implementations with&nbsp;**WebIDL2JS** in&nbsp;environments with&nbsp;IntelliSense¹&nbsp;easier, as&nbsp;it&nbsp;provides&nbsp;hints about&nbsp;what each&nbsp;function&nbsp;returns and&nbsp;takes as&nbsp;parameters.

---

¹ For&nbsp;example: [Visual&nbsp;Studio&nbsp;Code](https://code.visualstudio.com) or&nbsp;[NetBeans&nbsp;IDE](https://netbeans.apache.org/).

### CODE REVIEW

File: lib/output/utils.js [hunk 1]
```diff
@@ -89,27 +99,42 @@ const namedDelete = Symbol("named property delete");
 module.exports = exports = {
   isObject,
   hasOwn,
+  /** @type {typeof wrapperSymbol} */
```
Comment (ExE-Boss) [inline lib/output/utils.js]: This&nbsp;wouldn’t be&nbsp;necessary if&nbsp;we&nbsp;used the&nbsp;following&nbsp;instead:
```js
const wrapperSymbol = Symbol("wrapper");
exports.wrapperSymbol = wrapperSymbol;
```

<sup><sup><sub>This&nbsp;feels more&nbsp;like a&nbsp;quirk of&nbsp;**Salsa**&nbsp;though (**TypeScript**’s&nbsp;**JavaScript** JSDoc&nbsp;IntelliSense&nbsp;engine).</sub></sup></sup>

**Comment by domenic** [comment]
Thanks, but I'd prefer not to add these.

**Comment by ExE-Boss** [comment]
Can&nbsp;you&nbsp;provide me&nbsp;with your&nbsp;justification?

Because&nbsp;I&nbsp;believe that&nbsp;this&nbsp;would be&nbsp;a&nbsp;useful&nbsp;addition for&nbsp;contributors who&nbsp;use&nbsp;IDEs with&nbsp;IntelliSense.

**Comment by domenic** [comment]
I think the tradeoff of the noise in the source for all contributors and readers of the code is not worth the benefits of IntelliSense in some IDEs for some writers of the code.

**Pull request closed without merging**